components/admin_console/admin_definition.jsx

```diff
@@ -6303,6 +6425,72 @@ const AdminDefinition = {
                         isHidden: it.not(it.licensedForFeature('SAML')),
                         isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.EXPERIMENTAL.FEATURES)),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ServiceSettings.ThreadAutoFollow',
```

Comment:
It looks like this setting and the one below it have moved in https://github.com/mattermost/mattermost-webapp/pull/10465 since you started working on this, so they shouldn't be added back in. Can you actually remove this block of changes?